simulation/vtk_support.py

```diff
@@ -0,0 +1,138 @@
+"""Created by Prabhu Ramachandran in Feb. 2008."""
```

Comment:
This code appears to be [included in vtk](https://github.com/Kitware/VTK/blob/master/Wrapping/Python/vtkmodules/util/numpy_support.py).  Is there a reason you are copying it here?

Reply:
I believe the current version of VTK is 8.2.0 on GitHub, but the VTK on [PyPI](https://pypi.org/project/vtk/) is 8.1.2. In 8.1.2, it will throw a future warning 
```
FutureWarning: Conversion of the second argument of issubdtype from `complex` to `np.complexfloating` is deprecated. In future, it will be treated as `np.complex128 == np.dtype(complex).type`.
```
at 
```python
assert not numpy.issubdtype(z.dtype, complex)
```
which is fixed in 8.2.0
```python
assert not numpy.issubdtype(z.dtype, numpy.dtype(complex).type)
```

I am not an expert in Pipenv, but I think the VTK will be installed from PyPI while `pipenv install` is called. In this way, I think it will install 8.1.2 but not the latest 8.2.0, so I copied this piece of code from GitHub instead of using the original version from the PyPI package.

Reply:
Or we can simply suppress the warning if you think that is a better approach.

Reply:
I see.  Yes, I would ignore the warning for now.  We are freezing the numpy version so that bug won't hit us until we update.

Reply:
FYI, VTK will be back with updated versions on PyPI when they release 9.0. I agree it's best to leave the warning for now.

Reply:
FYI, VTK will be back with updated versions on PyPI when they release 9.0. I agree it's best to leave the warning for now.